lib/Basics/process-utils.cpp

```diff
@@ -1050,23 +1082,20 @@ bool TRI_WritePipe(ExternalProcess const* process, char const* buffer,
 
 ExternalProcessStatus TRI_CheckExternalProcess(ExternalId pid, bool wait,
                                                uint32_t timeout) {
-  ExternalProcessStatus status;
-  status._status = TRI_EXT_NOT_FOUND;
-
-  auto external = TRI_LookupSpawnedProcess(pid._pid);
+  auto status = TRI_LookupSpawnedProcessStatus(pid._pid);
 
-  if (external == nullptr) {
-    status._errorMessage =
+  if (!status.has_value()) {
+    status->_errorMessage =
```

Comment:
This will fail, because is `status` doesn't have a value, you cannot use `operator->` on it